### DISCUSSION TITLE
Web UI - archive.org icon shows in results list despite SAVE_ARCHIVE_DOT_ORG=False

### DISCUSSION THREAD

**Discussion by wanageeska** [ArchiveBox/ArchiveBox#1016] [Q&A]
(Disclaimer: I am a beginner and am not exactly sure if this is the right place to be asking these kinds of questions. If I'm in the wrong place please direct me to where I should be. If I am in the right place, please tell me what further info you would like me to provide.)

Question: Why is the [archive.org](https://web.archive.org/) icon displayed for every item in the results list even when `SAVE_ARCHIVE_DOT_ORG=False`?

I have confirmed that the setting is false via `archivebox config --get SAVE_ARCHIVE_DOT_ORG`  as per @pirate's instructions in this older thread: [Option to disable archiving to waybackmachine? #853](https://github.com/ArchiveBox/ArchiveBox/discussions/853). I have checked the logs to confirm that archivebox is definitely not sending to [archive.org](https://web.archive.org/). So why is that icon still included?

At first I thought it might have just indicated that the webpage in my local archive had also already been archived on [archive.org](https://web.archive.org/), which seemed reasonable since the icon, when clicked, links to the latest snapshot of that webpage stored on [archive.org](https://web.archive.org/).

But then I tested that by adding a local server page to archivebox and the icon still showed--and it _still_ redirected me to archive.org (though obviously throws a 404 because my local server page hasn't been saved to [archive.org](https://web.archive.org/)).

What am I missing here? Is it supposed to act as a little 'nudge' to users to also contribute to [archive.org](https://web.archive.org/)? Could someone please provide some guidance? I've spent FAR too much time trying to figure it out.

Thanks!

**Comment by pirate** [comment]
The icon is always shown because the SAVE_ARCHIVE_DOT_ORG setting can be toggled frequently / for individual runs / etc., so because it could be present on previous snapshots that were saved when the setting used to be true, we show it all the time. The same is true for all the extractor toggles, all icons are shown regardless of the toggle state because the state can be toggled per-archive, it's not a system-wide setting.

**Comment by wanageeska** [comment]
Hi @pirate, thanks for such a quick response!
I'm still confused, but rather than try to articulate what I'm seeing I think it might have been wiser just to attach a screenshot. Kinda like the one attached to this message... 😒 
If I'm understanding you correctly, you're referring to the greyed-out icons for the disabled extractors, right? What's confusing me is why the archive-org icon is still 'live' for the last snapshot of localhost, even though every extractor toggle is set to false.
It is obviously far, far, far from a big deal, and I'm sorry for being a pest, but I'm the type to flick all the switches to see what happens and that was something that didn't seem to change no matter what I flicked.

![Screenshot from 2022-08-30 16-35-39](https://user-images.githubusercontent.com/17569679/187369543-1a09c538-aa0d-4c6f-88c8-6a7e7bcdeebf.png)
